src/components/EditMinigameModals/EditMemoryConfigurationModal.vue

```diff
@@ -0,0 +1,414 @@
+<script setup lang="ts">
+import { saveAs } from "file-saver";
+import { arrayOf, defaultValue, int, object, oneOf, string } from "checkeasy";
+import { importConfiguration } from "@/ts/import-configuration";
+import { Ref, defineEmits, defineProps, onMounted, ref, watch } from "vue";
```

Comment:
```suggestion
import { arrayOf, object, oneOf, string } from "checkeasy";
import { importConfiguration } from "@/ts/import-configuration";
import { Ref, defineEmits, defineProps, ref, watch } from "vue";
```
remove unused imports